### PR TITLE
Add mobile lock icon and fix single select floating header

### PR DESCRIPTION
**Pull request by aoimasu**
- remove z-index of label in select
- add mobile lock icon

### CODE REVIEW

**Comment by hen-zone** [comment]
Hey @aoimasu, what does this change do? Is there a bug currently which this fixes?

**Comment by aoimasu** [comment]
Yes @hen-zone , the floating label floats on top of the select menu, this will fix that

**Comment by xow** [comment]
z-index is always terrifying 😱

File: src/components/Form/DatePickers/shared.module.scss [hunk 1]
```diff
@@ -115,7 +115,7 @@
       display: block
     }
     .SingleDatePicker_picker {
-      z-index: 1;
```
Comment (xow) [inline src/components/Form/DatePickers/shared.module.scss]: This now causes the arrow to be lower than the date picker. The speech bubble effect no longer looks good

File: src/components/Form/SingleSelect/SingleSelect.scss [hunk 2]
```diff
@@ -220,6 +218,7 @@
   font-size: $input-size;
   font-weight: $regular;
   overflow: hidden;
+  z-index: 3;
```
Comment (xow) [inline src/components/Form/SingleSelect/SingleSelect.scss]: We should just use sed to remove all lines using z-index and start again from scratch

File: src/components/Icon/Lock.tsx [hunk 3]
```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+
+const Lock: React.SFC<React.SVGProps<SVGSVGElement>> = props => (
```
Comment (xow) [inline src/components/Icon/Lock.tsx]: Has this been put through SVG go? or is that no longer a thing

Comment (aoimasu) [reply src/components/Icon/Lock.tsx]: yes, i put it through SVGGo alrady

**Pull request merged**